### PR TITLE
Add io.github.lo2dev.Echo

### PR DESCRIPTION
**Pull request by lo2dev**
Echo is a ping utility for desktop made with GTK4 and Libadwaita.
Network perm is used to reach domains.

### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: io.github.lo2dev.Echo.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "modules" : [
+    	"python3-icmplib.json",
+        {
+            "name" : "echo",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "dir",
+                    "path" : ".."
```
Comment (hfiguiere) [inline io.github.lo2dev.Echo.json:34]: this is not gonna work. it is not gonna find the source.

File: io.github.lo2dev.Echo.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	"python3-icmplib.json",
```
Comment (hfiguiere) [inline io.github.lo2dev.Echo.json:26]: this dosn't exist.

File: io.github.lo2dev.Echo.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            "name" : "echo",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/lo2dev/Echo.git"
```
Comment (bbhtt) [inline io.github.lo2dev.Echo.json:34]: You need a tag or a commit or both.

Comment (bbhtt) [reply io.github.lo2dev.Echo.json:34]: https://docs.flatpak.org/en/latest/module-sources.html#git-sources

**Comment by bbhtt** [comment]
bot, build io.github.lo2dev.Echo

File: io.github.lo2dev.Echo.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    	"python3-icmplib.json",
+        {
+            "name" : "echo",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/lo2dev/Echo.git",
+                    "tag": "v1.0",
```
Comment (bbhtt) [inline io.github.lo2dev.Echo.json:35]: Tag does not exist.

**Comment by lo2dev** [comment]
Apologies, pushed the tag to remote hopefully it works

**Comment by bbhtt** [comment]
bot, build io.github.lo2dev.Echo

**Comment by lo2dev** [comment]
Not sure if the fail is related to my repo, but there are some things i have to fix upstream before finalizing this if that's okay

**Comment by bbhtt** [comment]
You are missing the blueprint-compiler dependency in the manifest.

**Comment by bbhtt** [comment]
bot, build io.github.lo2dev.Echo

**Pull request closed without merging**